site/src/pages/CreateWorkspacePage/CreateWorkspacePageView.tsx

```diff
@@ -256,7 +256,7 @@ export const CreateWorkspacePageView: FC<CreateWorkspacePageViewProps> = ({
 				<FormSection
 					title="General"
 					description={
-						permissions.createWorkspaceForUser
+						permissions.createWorkspace
 							? "The name of the workspace and its owner. Only admins can create workspaces for other users."
 							: "The name of your new workspace."
```

Comment:
This is now incorrect. The original `createWorkspaceForUser` was correct here. If that was false, the text would just be `"The name of your new workspace."`